### PR TITLE
Fix typo in external-apps-ingress-controller.yaml

### PR DESCRIPTION
**Pull request by dystewart**
In response to [issue](https://github.com/OCP-on-NERC/operations/issues/29)

### CODE REVIEW

**Review by joachimweyl — APPROVED** [review]

**Review by hpdempsey — APPROVED** [review]
Looks good

File: cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps-ingress-controller.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: external-apps-ingress-controller
   namespace: openshift-ingress-operator
 spec:
-  domain: apps.shift.nerc.mghpcc.org
+  domain: apps.openshift.nerc.mghpcc.org
```
Comment (larsks) [inline cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps-ingress-controller.yaml:7]: What is the motivation for this change? The original value was correct:

```
$ host foo.apps.shift.nerc.mghpcc.org
foo.apps.shift.nerc.mghpcc.org has address 199.94.61.6
$ host foo.apps.openshift.nerc.mghpcc.org
Host foo.apps.openshift.nerc.mghpcc.org not found: 3(NXDOMAIN)
```

**Review by larsks — CHANGES_REQUESTED** [review]

File: cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps-ingress-controller.yaml (hunk 1 above)
Comment (dystewart) [reply cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps-ingress-controller.yaml:7]: Ah I thought it may have been a typo, disregard
